### PR TITLE
Added gitapp.si

### PR DESCRIPTION
**Pull request by aljaxus**
Added gitapp.si

* [x] Description of Organization
* [x] Reason for PSL Inclusion
* [x] DNS verification via dig
* [x] Run Syntax Checker (make test)

Description of Organization
====

Gitplac.si is a privately owned self hosted Gitlab instance. The platform is currently closed for new signups, but is planned to open in near future. 

Organization Website:
https://gitplac.si

Reason for PSL Inclusion
====
Cookie Security, Let's Encrypt issuance

The domain will be used to automatic application deployments. Any user will be able to deploy an application, manage their cookies, request TLS certificates for new subdomains, ...

DNS Verification via dig
=======

```
[aljaxus@desktop ~]$ drill _psl.gitapp.si TXT
;; ->>HEADER<<- opcode: QUERY, rcode: NOERROR, id: 62854
;; flags: qr rd ra ; QUERY: 1, ANSWER: 1, AUTHORITY: 0, ADDITIONAL: 0 
;; QUESTION SECTION:
;; _psl.gitapp.si.      IN      TXT

;; ANSWER SECTION:
_psl.gitapp.si. 60      IN      TXT     "https://github.com/publicsuffix/list/pull/1091"

;; AUTHORITY SECTION:

;; ADDITIONAL SECTION:

;; Query time: 67 msec
;; SERVER: 1.0.0.1
;; WHEN: Wed Aug 26 18:42:51 2020
;; MSG SIZE  rcvd: 105
[aljaxus@desktop ~]$ 

```

make test
=========

```
============================================================================
Testsuite summary for libpsl 0.21.1
============================================================================
# TOTAL: 5
# PASS:  5
# SKIP:  0
# XFAIL: 0
# FAIL:  0
# XPASS: 0
# ERROR: 0
============================================================================
Making check in msvc
[aljaxus@desktop list]$ 
```

### CODE REVIEW

**Comment by dnsguru** [comment]
Expands #1013

File: public_suffix_list.dat [hunk 1]
```diff
@@ -11859,6 +11859,7 @@ gitlab.io
 // Gitplac.si - https://gitplac.si
 // Submitted by Aljaž Starc <me@aljaxus.eu>
 gitpage.si
+gitapp.si
```
Comment (dnsguru) [inline public_suffix_list.dat:11862]: move above gitpage.si so that the gitplac section domains are alphebetic order by sld

**Comment by aljaxus** [comment]
> Sorry to be pedantic, but please reverse the order of these two entries so that they are in alphabetic order.

Oh yeah, I'm sorry. My bad, no need to apologize.

**Review by dnsguru — APPROVED** [review]
Thank you for alphabetizing; 

* DNS Test validates
* no conflict with base
* tests pass
* rationale good

**APPROVED**

**Comment by dnsguru** [comment]
@aljaxus MERGED

**Pull request merged**